index.html

```diff
@@ -0,0 +1,195 @@
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="social-media">
+      <div class="hidden">
+        <p>work</p>
+      </div>
+      <div>
+        <ul class="media-icons">
+          <li>
+            <a href="#">
+              <img src="./images/github.png" alt="image" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./images/Vector.png" alt="image" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./images/Vector2.png" alt="image" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./images/twitter.png" alt="image" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./images/Vector3.png" alt="image" />
+            </a>
+          </li>
+        </ul>
+        <div class="work-below">
+          <img src="./images/down-arrow.png" alt="" /> <span>work</span>
+        </div>
+      </div>
+      <div class="hidden">
+        <p>work</p>
+      </div>
+    </div>
+  <!--Part 2 DAY 4.....Portfolio webpage-->
+  <section class="recent-works">
+    <div class="recent-work-container">
+      <div class="my-works">
+        <h2> RECENT WORK DONE</h2>
+        <div class="my-work-contain">
+          <div class="my-work-card">
+            <img src="./images/ImgPlaceholder.png" alt="image" />
+            <h3 class="title">Multy-Post Stories</h3>
+            <p class="work-description">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required. has been the industry's standard dummy
+              text ever since the 1500s, when an unknown printer took a
+              standard dummy text.
+            </p>
+            <ul class="prog-langs">
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button type="button" class="btn">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="recent-work-container1">
+      <div class="my-works">
+        <div class="my-work-contain">
+          <div class="my-work-card">
+            <img src="./images/ImgPlaceholder.png" alt="image" />
+            <h3 class="title">Multy-Post Stories</h3>
+            <p class="work-description">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required. has been the industry's standard dummy
+              text ever since the 1500s, when an unknown printer took a
+              standard dummy text.
+            </p>
+            <ul class="prog-langs">
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
```

Comment:
- [ ] Kindly remove this horizontal line before CSS and use that CSS li at left side
### your design
![his](https://user-images.githubusercontent.com/81727988/178110191-72ab7c32-e726-4ffa-809e-aef8a265d1e6.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/178110202-7b063718-2915-4498-9452-41fb9c7268ed.png)